CSS/Components/card.css

```diff
@@ -0,0 +1,87 @@
+.card-container {
+  width: 80%;
+  padding: var(--sp-md);
+  background-color: var(--gray-100);
+  box-shadow: 0 0 2px var(--gray-700);
+}
+.card {
+  width: 20rem;
+  border: 1px solid var(--gray);
+  position: relative;
+}
+.card__img {
+  width: 100%;
+  object-fit: cover;
+}
+.card-title,
+.card__price {
+  font-weight: var(--fw-semibold);
+}
+.card-desc {
+  color: var(--gray);
+  font-size: small;
+}
+.card__price {
+  gap: var(--sp-xsm);
+}
+.card__price--offer {
+  color: var(--primary);
+}
+.card__badge {
+  border-radius: 5%;
+  padding: 0.2rem 0.4rem;
```

Comment:
for values less than 10px, you can use px instead of rems to make it more readable

Reply:
Thanks for suggestion, Updated the same.